locksmith/src/controllers/v2/ticketsController.tsx

```diff
@@ -57,15 +57,16 @@ export class TicketsController {
       })
 
       const data = keyMetadata?.data as unknown as {
-        metadata: { checkedInAt: number }
+        metadata: { checkedInAt: number | number[] }
       }
 
-      const isCheckedIn = data?.metadata?.checkedInAt
-
-      if (isCheckedIn) {
-        return response.status(409).send({
-          error: 'Ticket already checked in',
-        })
+      const checkedInAt = []
+      if (!data?.metadata?.checkedInAt) {
+        checkedInAt.push(new Date().getTime())
+      } else if (typeof data?.metadata?.checkedInAt === 'number') {
+        checkedInAt.push(data?.metadata?.checkedInAt, new Date().getTime())
+      } else if (Array.isArray(data?.metadata?.checkedInAt)) {
+        checkedInAt.push(...data.metadata.checkedInAt, new Date().getTime())
       }
```

Comment:
can you add some tests for this part specifically in the endpoint?

Reply:
Ha good point! Let me add that a bit later as I need to merge this ASAP